modules/codeready-workspaces.adoc

```diff
@@ -0,0 +1,9 @@
+
+[id="codeready-workspaces_context}"]
```

Comment:
Ah, didn't catch this before and it looks like it's causing the build to fail. Needs to be `codeready-workspaces_{context}`.